pkg/operator/k8sutil/configmap.go

```diff
@@ -36,3 +39,38 @@ func DeleteConfigMap(clientset kubernetes.Interface, cmName, namespace string, o
 	defaultWaitOptions := &WaitOptions{RetryCount: 20, RetryInterval: 2 * time.Second}
 	return DeleteResource(delete, verify, resource, opts, defaultWaitOptions)
 }
+
+// getConfigMap returns a configMap or error
+func getConfigMap(clientset kubernetes.Interface, cmName, namespace string) (*corev1.ConfigMap, error) {
```

Comment:
This is a small enough method I'd suggest just merging it wtih `GetOperatorSettings()`